### PR TITLE
Add block_cluster_admin_rolebindingquery for Kubernetes 

### LINKED ISSUE #645: Add block_cluster_admin_rolebindingquery for Kubernetes
### Platform
*Kubernetes*

### Description
*Ensure that the cluster-admin role is only used where required (RBAC)*

### PR DESCRIPTION
**Pull request by rogeriopeixotocx**
Ensure that the cluster-admin role is only used where required (RBAC)

Closes #645

### CODE REVIEW

File: assets/queries/k8s/block_cluster_admin_rolebinding/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "block_cluster_admin_rolebinding",
+  "queryName": "Block Cluster Admin Rolebinding",
+  "severity": "LOW",
+  "category": null,
+  "descriptionText": "Ensure that the cluster-admin role is only used where required (RBAC)",
+  "descriptionUrl": "#"
```
Comment (jorge-cruz) [inline assets/queries/k8s/block_cluster_admin_rolebinding/metadata.json:7]: Add descriptionUrl https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles

File: assets/queries/k8s/block_cluster_admin_rolebinding/query.rego [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  resource := input.document[i]
+  metadata := resource.metadata
+  resource.kind == "ClusterRoleBinding"
+  resource.roleRef.name == "cluster-admin"
+
+	result := {
+    "documentId": input.document[i].id,
+    "searchKey": sprintf("metadata.name=%s.roleRef.name=cluster-admin", [metadata.name]),
+    "issueType": "IncorrectValue",
+    "keyExpectedValue": "Ensure that the cluster-admin role is only used where required (RBAC)",
```
Comment (jorge-cruz) [inline assets/queries/k8s/block_cluster_admin_rolebinding/query.rego:13]: Change keyExpectedValue to:
`sprintf("Resource name '%s' of kind '%s' isn't binding 'cluster-admin' role with superuser permissions", [metadata.name, resource.kind])`

**Review by jorge-cruz — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/block_cluster_admin_rolebinding/metadata.json (hunk 1 above)
Comment (rogeriopeixotocx) [reply assets/queries/k8s/block_cluster_admin_rolebinding/metadata.json:7]: done!

File: assets/queries/k8s/block_cluster_admin_rolebinding/query.rego (hunk 2 above)
Comment (rogeriopeixotocx) [reply assets/queries/k8s/block_cluster_admin_rolebinding/query.rego:13]: ok!

File: assets/queries/k8s/block_cluster_admin_rolebinding/metadata.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "block_cluster_admin_rolebinding",
+  "queryName": "Block Cluster Admin Rolebinding",
+  "severity": "LOW",
+  "category": null,
+  "descriptionText": "Ensure that the cluster-admin role is only used where required (RBAC)",
+  "descriptionUrl": "feature/block_cluster_admin_rolebinding"
```
Comment (jorge-cruz) [inline assets/queries/k8s/block_cluster_admin_rolebinding/metadata.json:7]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles
Still missing the URL :stuck_out_tongue_winking_eye:

**Review by jorge-cruz — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/block_cluster_admin_rolebinding/metadata.json (hunk 3 above)
Comment (rogeriopeixotocx) [reply assets/queries/k8s/block_cluster_admin_rolebinding/metadata.json]: Sorry :P

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**